go/bind/notifications.go

```diff
@@ -34,11 +35,11 @@ type Message struct {
 }
 
 type ChatNotification struct {
-	// Ordered from oldest to newest
 	Message   *Message
 	ConvID    string
 	TeamName  string
 	TopicName string
+	TlFName   string
```

Comment:
Does this field name pass `go vet`?

Reply:
Apparently! Want me to change it?

Reply:
It looks like it was probably a typo, so I think we should.

Reply:
Oh, I see the typo now